src/hotspot/share/classfile/classLoader.cpp

```diff
@@ -1415,6 +1415,23 @@ char* ClassLoader::lookup_vm_options() {
   return options;
 }
 
+// Returns true if jdk.internal.vm.ci is present on the file system.
+bool ClassLoader::has_jvmci_module() {
```

Comment:
Would it be more useful to pass the module name so that the function tests if the module is is in the run-time image so that ClassLoader doesn't need to know the name "jdk.internal.vm.ci"?